solidity/contracts/converter/ConverterBase.sol

```diff
@@ -258,6 +258,18 @@ abstract contract ConverterBase is ConverterVersion, IConverter, ContractRegistr
         acceptOwnership();
     }
 
+    /**
+     * @dev does nothing
```

Comment:
"executed by the upgrader at the end of the upgrade process to handle and custom pool logic"

Reply:
And in the StandardPoolConverter?

Reply:
oops, had a typo there ("and")

Reply:
and can use it anywhere

Reply:
Done